torch/nn/modules/sparse.py

```diff
@@ -6,6 +6,7 @@
 from .. import init
 
 
+@torch._jit_internal.weak_module
```

Comment:
Shall we keep the same semantics when annotating modules? I am seeing different ways of doing it: 1. `from torch._jit_internal import weak_module, weak_script_method` 2. directly annotate them with `torch._jit_internal.weak_module`.. I think we better keep one syntax. CC @driazati

Reply:
Ya good point. I guess I would prefer the first, because it's kind of annoying to expose torch.jit & torch._jit_internal